usr.bin/truss/Makefile

```diff
@@ -9,4 +9,6 @@ LIBADD=	sysdecode
 #CFLAGS+= -I${.CURDIR} -I. -I${SRCTOP}/sys
 CFLAGS+= -I${SRCTOP}/sys
 
+_CHERI_CFLAGS+= -cheri-uintcap=addr
```

Comment:
I'm not sure if this is the right way to do this?

Reply:
Probably fine for now. I guess we could use a CHERI_UINTCAP_MODE variable and change bsd.cheri.mk to pass `-cheri-uintcap=${CHERI_UINTCAP_MODE:Uoffset}` (not sure about the make syntax for a default value though).

Reply:
I added CHERI_UINTCAP_MODE.